conf.py

```diff
@@ -160,6 +161,9 @@
     target_locale="" if language == "en" else "/" + language,
 )
 
+# Intersphinx for classref links in localized docs.
+intersphinx_mapping = {"api": ("https://docs.godotengine.org/en/{}".format(version), None)}
```

Comment:
I don't like the fact that even on local builds and on the English docs, the classref links will go through this intersphinx mapping instead of keeping a local reference.

But so far I couldn't find documentation on how to do things in a better way. It should be either:
- An intersphinx feature to provide a "self" reference when `not is_i18n` (couldn't find any reference to such feature).
- A sphinx feature to let us define a dummy `api` cross-reference role similar to the one intersphinx would define, but which would do nothing on `not is_i18n`. Couldn't find any reference to that either.
- Possibly doable, but needs research: defining a custom `:api:` role that would convert the reference based on whether we are `is_i18n` or not. That would be the best solution as we could then keep links as <pre>:api:`class_node`</pre> instead of <pre>:ref:`api:class_node`</pre>, but the docs for creating custom rst roles is not immediately obvious to me: https://docutils.sourceforge.io/docs/howto/rst-roles.html

Reply:
I tried to implement the last option, but I'm stuck. I can't find a way to register a role that simply wraps `:api:` to `:ref:`, I'd have to resolve the `class_node` label myself to find out the URI of the page...

If anyone wants to dig into that stuff: https://github.com/akien-mga/godot-docs/tree/test-sphinx-role-api